Add Card class

daelynj opened the pull request:
Add a Card class to hold the suit and rank of a card with a method to
print a card's suit and rank.

daelynj committed:
Add Card class

Add a Card class to hold the suit and rank of a card with a method to
print a card's suit and rank.

jgayfer commented on lib/poker_hands/card.rb:
```diff
@@ -0,0 +1,10 @@
+        @suit = suit
+    end
+
+    def get_card
```
Lots of objects in Ruby have a builtin `to_s` (to string) method, which is used for getting a string representation of the object.

If we wanted to adhere to that convention, we could name this method `to_s`

```
> card = Card.new(1, 'H')
> card.to_s
"1H"
```

This would also help communicate that we can expect the method to return a string.

jgayfer commented on lib/poker_hands/card.rb:
```diff
@@ -0,0 +1,10 @@
+    def get_card
+        @rank + @suit
+    end
+end
```
By convention, we usually use a 2 space indentation (as opposed to 4).

http://www.caliban.org/ruby/rubyguide.shtml#indentation

Github is also complaining about a lack of newline at the end of the file, I believe you
can fix that with your editor 👍

jgayfer reviewed (commented):
Congrats on your first PR! 🎉 

+1 on the commit message, appreciate the detail.

daelynj committed:
Use 2 space indentation

Ruby uses 2 space indentation by convention.

daelynj committed:
Add newline at end of file

Github expects a newline at end of file.

daelynj committed:
Override built-in to_s

Override built-in to_s to combine a Card objects rank and suit
to get a string representation of the object.
